src/Pact/Native.hs

```diff
@@ -1338,9 +1342,12 @@ strToInt i as =
     doBase si base txt = case baseStrToInt base txt of
       Left e -> evalError' si (pretty e)
       Right n -> return (toTerm n)
-    doBase64 si txt = case parseB64UrlUnpaddedText' txt of
-      Left e -> evalError' si (pretty e)
-      Right bs -> return $ toTerm $ bsToInteger bs
+    doBase64 si txt = do
+      base64Behavior <- (\legacy -> if legacy then Legacy else Simplified) <$> isExecutionFlagSet FlagDisablePact49
+      parseResult <- base64DecodeWithShimmedErrors (getInfo si) base64Behavior txt
+      case parseResult of
+        Left e -> evalError' si (pretty e)
```

Comment:
is `pretty` here correct?

Reply:
I believe so. I was trying to replicate the old behavior, which used `pretty`. And `evalError'` takes a `Doc` (the output of `pretty`). But, do you see an issue?

Reply:
(I made a note on the other comment you left about `pretty`, resolving this one).